templates/support/community-support.html

```diff
@@ -29,7 +29,7 @@ <h1>Community support</h1>
 <div class="p-strip is-deep">
   <div class="row u-equal-height">
     <div class="col-4 p-card">
-      <h3><a href="https://discourse.ubuntu.com/t/community-support/709">Find community support</a></h3>
+      <h3><a href="https://ubuntu.com/community/support">Find community support</a></h3>
       <p><a href="https://askubuntu.com/">Ask Ubuntu</a> and <a href="https://ubuntuforums.org/">Ubuntu Forums</a> are great sources of answers to common questions about installing, troubleshooting and optimising Ubuntu. For more info on support forums, including IRC channels, see <a href="https://discourse.ubuntu.com/t/community-support/709">our Discourse site</a>.</p>
```

Comment:
This still uses the old discourse link, did you want to update this as well? The copy here would also need to change as it would no longer be routing directly to discourse

Reply:
Yes, that link should be updated as well. Thank you, mtruj013!